py/dml/compat.py

```diff
@@ -130,7 +130,16 @@ class suppress_WLOGMIXUP(CompatFeature):
     to avoid overwhelming users with warnings, as the faulty pattern that
     `WLOGMIXUP` reports is very prevalent within existing code. Addressing
     applications of the faulty pattern should be done before or as part of
-    migration to Simics API version 7.'''
+    migration to Simics API version 7.
+
+    Passing `--no-compat=suppress_WLOGMIXUP` to DMLC has almost the same effect
+    as passing `--warn=WLOGMIXUP`; either will cause DMLC to report the warning
+    even when the Simics API version in use is below 7. The only difference
+    between these two options is that if `--no-compat=suppress_WLOGMIXUP` is
+    used (and `--warn=WLOGMIXUP` is not), then `WLOGMIXUP` may still be
+    explicitly suppressed via `--no-warn=WLOGMIXUP`. In contrast,
+    `--warn=WLOGMIXUP` doesn't allow for `WLOGMIXUP` to be suppressed at
+    all.'''
```

Comment:
@MagnusIntel @mandolaerik Does this look fine?

Reply:
I find it unlikely that anyone would consume such fine details, but at least it is complete